_includes/com-config/com-config-bind-extra.md

```diff
@@ -0,0 +1,4 @@
+### `bind` additional
```

Comment:
Only used once. Does this have a plan to be re-used?

Reply:
Yep, that'll feature in a new branch/pr explaining how to create custom config files and to run FeatureBase so it uses them instead of the one supplied with the install.